docs/monitoring.md

```diff
@@ -220,6 +220,13 @@ The history server can be configured as follows:
       Number of threads that will be used by history server to process event logs.
     </td>
   </tr>
+  <tr>
+    <td>spark.history.store.path</td>
```

Comment:
Check from the log, looks like we don't have a default path for local history store, do we need to add a default value here in doc?

```  
val LOCAL_STORE_DIR = ConfigBuilder("spark.history.store.path")
    .stringConf
    .createOptional
```